doc/adr/0020-extending-amended-expressions-for-orphan-removal-process.md

```diff
@@ -1,53 +1,81 @@
+| true                    | true                                       | Expressions created by _zeitgrenze_ that also replace an existing one (use case 2).   |
 
 Example:
+
 ```
 <norms:legalDocML.de_metadaten xmlns:norms="http://MetadatenMods.LegalDocML.de/1.8.1/">
     <norms:amended-norm-expressions>
-        <norms:norm-expression creation="NEW">eli/bund/bgbl-1/1964/s593/2017-03-16/1/deu</norms:norm-expression>
-        <norms:norm-expression creation="SYSTEM">eli/bund/bgbl-1/1964/s593/2021-01-01/2/deu</norms:norm-expression>
-        <norms:norm-expression creation="REPLACING">eli/bund/bgbl-1/1964/s593/2022-02-02/2/deu</norms:norm-expression>
+        <!-- New "normal" expression -->
+        <norms:norm-expression created-by-zeitgrenze="true" created-by-replacing-existing-expression="false">eli/bund/bgbl-1/1964/s593/2017-03-16/1/deu</norms:norm-expression>
+        <!-- Created by SYSTEM because already existing expression-->
+        <norms:norm-expression created-by-zeitgrenze="false" created-by-replacing-existing-expression="true">eli/bund/bgbl-1/1964/s593/2021-01-01/2/deu</norms:norm-expression>
+        <!-- New "normal" expression -->
+        <norms:norm-expression created-by-zeitgrenze="true" created-by-replacing-existing-expression="false">eli/bund/bgbl-1/1964/s593/2022-02-02/2/deu</norms:norm-expression>
+        <!-- Created because of time boundary AND replacing existing expression-->
+        <norms:norm-expression created-by-zeitgrenze="true" created-by-replacing-existing-expression="true">eli/bund/bgbl-1/1964/s593/2022-02-02/2/deu</norms:norm-expression>
     </norms:amended-norm-expressions>
 </norms:legalDocML.de_metadaten>
 ```
 
-Using this attribute in the algorithm for orphan removal would work like this:
+Using these attributes in the algorithm for orphan removal process, we would iterate over
+`norms:amended-norm-expressions` from the oldest to the most recent, performing the following steps:
 
-- NEW: remove the orphan if the _zielnorm_ reference is gone (current state)
-- SYSTEM: remove the orphan only if there are no newly created expressions before (within `norms:amended-norm-expressions`)
-- REPLACING: remove the orphan only if the _zielnorm_ reference is gone AND there is no newly created expressions before (meaning combining conditions of NEW and SYSTEM)
+1. If `created-by-zeitgrenze=true` and `created-by-replacing-existing-expression=false` and the _zeitgrenze_ is no
+   longer present under the _zielnorm_ references, the entry is deleted
+2. If `created-by-zeitgrenze=false` and `created-by-replacing-existing-expression=true` and there are no preceding
+   expressions with `created-by-zeitgrenze=true`, the entry is deleted
+3. If `created-by-zeitgrenze=true` and `created-by-replacing-existing-expression=true` and neither the _zeitgrenze_ is
+   present under the _zielnorm_ references nor are there preceding expressions with `created-by-zeitgrenze=true`, the
+   entry is deleted
```

Comment:
4. If `created-by-zeitgrenze=true` and `created-by-replacing-existing-expression=true` and the _zeitgrenze_ is not present under the _zielnorm_ references but there are there preceding expressions with `created-by-zeitgrenze=true`, the entry is changed to `created-by-zeitgrenze=false` and `created-by-replacing-existing-expression=true`

Reply:
Agree with 4. but definitely not with 5. If we still have the _zeitgrenze_ we don't really care if there are preceding expressions or not. Since that expression is overriding still the already existing expression